importer/RoRo.py

```diff
@@ -1,29 +1,117 @@
-from Monument import Monument
+from Monument import Monument, Dataset
 import importer_utils as utils
+import importer as importer
 
-class RoRo(Monument):
 
-    def update_labels(self):
-        name = utils.remove_markup(self.denumire)
-        self.add_label("ro", name)
+class RoRo(Monument):
 
     def set_adm_location(self):
-        return
+        counties = self.data_files["counties"]
```

Comment:
You can replace all of this with the `self.set_from_dict_match()` notation

Reply:
Done.